.github/dependabot.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 updates:
-  - package-ecosystem: "github-actions"
+  - package-ecosystem: "gradle"
     directory: "/"
```

Comment:
no, keep actions, but add gradle as another - package ecosystem below

github-actions is used to update /.github/workflows/*.yaml so that you don't end up getting stuck using old versions of things like "actions/checkout@v4" in there since they update/deprecate them pretty frequently